securedrop_client/gui/widgets.py

```diff
@@ -2270,8 +2270,8 @@ def _on_export_clicked(self):
         if not self.controller.downloaded_file_exists(self.file):
             return
 
-        dialog = ExportDialog(self.controller, self.uuid, self.file.filename)
-        dialog.exec()
+        self.export_dialog = ExportDialog(self.controller, self.uuid, self.file.filename)
+        self.export_dialog.show()
```

Comment:
This works fine since the dialogs make sure to not allow multiple dialogs at the same time.